.docker/compose.controller.yaml

```diff
@@ -39,7 +39,7 @@ services:
     networks:
       - vrm
```

Comment:
not a big deal but we should change the network name to specific to this proj, suggestion "air"